src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskSearchControllerTest.java

```diff
@@ -1708,7 +1708,7 @@ void should_return_status_code_502_when_idam_service_is_down() throws Exception
                 jsonPath("$.status").value(502),
                 jsonPath("$.detail").value(
                     "Downstream dependency did not respond as expected "
-                    + "and the request could not be completed.")
+                    + "and the request could not be completed. Message from downstream system: null")
```

Comment:
How come this is still null?

Reply:
I missed these. I will fix all the null values.